bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/filter/core/FilterXMLFormatter.scala

```diff
@@ -28,9 +28,9 @@ object FilterXMLFormatter {
     <qvTool>
       <filters>{filters.map(format)}</filters>
       <axes>{axes.map(format)}</axes>
-      <charts>{histograms.map(_.toXml)}</charts>
+      <histograms>{histograms.map(_.toXml)}</histograms>
       <tables>{tables.map(_.toXml)}</tables>
-      <visCharts>{barCharts.map(_.toXml)}</visCharts>
```

Comment:
I'm quite confident that you've fixed the problem, and it looks like more of the same so I'm also sure it's fine.  Still I feel a bit sheepish giving the thumbs up as if I knew what is going on.  Just for my own edification a word or two about what the difference between, for example, labeling this "barCharts" vs "visCharts" actually does would be great.